opts.go

```diff
@@ -0,0 +1,23 @@
+package header
+
+type RequestOption func(opts *RequestParams)
```

Comment:
If it's only for Head, this should be HeadOption

Reply:
done